lib/api/multiObjectDelete.js

```diff
@@ -480,9 +480,12 @@ function multiObjectDelete(authInfo, request, log, callback) {
         }
         const xml = _formatXML(quietSetting, errorResults,
             successfullyDeleted);
+        const deletedKeys = [];
+        successfullyDeleted.map(item => deletedKeys.push(item.key));
```

Comment:
you can do this, no need to create a separate array outside:
`const deletedKeys = successfullyDeleted.map(item => item.key);`